python/tensorflow-load-library.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: tensorflow-load-library
+  patterns:
+    - pattern-either: 
+      - pattern: |
+      tf.load_library($PATH)
```

Comment:
I would replace "tf" with "tensorflow" to generalize across different instantiations of the name.